qiskit/max-cut/maxCut_qiskit.py

```diff
@@ -31,6 +31,25 @@
 input_path = sys.argv[1]
 output_path = sys.argv[2]
 
+# To include the weight information, we'll need to map the weight to the label
+# so the parser can read it. Therefore, remove all existing "label" lines and
```

Comment:
does this code really remove label lines? where?

Reply:
Yes, it does so by skipping the lines with "label" in it in the if down below.